plugin/trino-phoenix-5/src/test/java/io/trino/plugin/phoenix/TestingPhoenixServer.java

```diff
@@ -91,6 +91,8 @@ private TestingPhoenixServer()
         this.conf.setInt(REGIONSERVER_INFO_PORT, -1);
         this.conf.setInt(HBASE_CLIENT_RETRIES_NUMBER, 1);
         this.conf.setBoolean("phoenix.schema.isNamespaceMappingEnabled", true);
+        // needed to work around shading of native libraries in HBase
+        this.conf.setBoolean("hbase.netty.nativetransport", false);
```

Comment:
Can you please make  this a TODO comment with a link to a Trino issue describing the problem?

Reply:
Yes.

Note that this is a test-only problem, though, since here we are actually running an HBase _server_ so that we can run the tests.

Before I do that I will also double check whether with the latest binary artifacts of Phoenix and HBase it is still a problem.

Reply:
I can still run all my tests locally without this change, so the shading issues where solved in HBase. I'm just going to remove this change.